manifests/m/mercedesclk320/SamsungRemote/1.0/mercedesclk320.SamsungRemote.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: mercedesclk320.SamsungRemote
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: mercedes.clk320
+PackageName: samsung remote control app

```

Comment:
This should be Title Cased at minimum. If possible, a better name should be used since this is a portable app